c2_utils/src/mfx_c2_utils.cpp

```diff
@@ -373,7 +373,7 @@ static const std::pair<C2Config::level_t, mfxU16> g_h264_levels[] =
 static const std::pair<C2Config::profile_t, mfxU16> g_h265_profiles[] =
 {
     { PROFILE_HEVC_MAIN,  MFX_PROFILE_HEVC_MAIN },
-    { PROFILE_HEVC_MAIN_STILL, MFX_PROFILE_HEVC_MAINSP },
```

Comment:
We should support HEVC_MAIN_STILL profile. Why to remove it?

Reply:
Because there is a complex logic to check whether it should be used. 
I haven't figure it out yet. Once I figure it out, I'll support it again.

https://github.com/projectceladon/oneVPL-Intel-GPU/blob/3c91404a434a94fc6ffedadffbd5ec9c2355d8c1/_studio/mfx_lib/encode_hw/hevc/agnostic/base/hevcehw_base_legacy_defaults.cpp#L2456